Dialogue/Alma/co_050049.csv

```diff
@@ -1,3 +1,3 @@
 name01#0,"""Alma"""
-text01#0,"""Hey <%me>! Could we view the new archive?"""
+text01#0,"""Hey, <%me>! Could we view the new archive?"""
```

Comment:
There's not supposed to be a pause.

Reply:
NPC is directly addressing the player. Commas are used with direct address.